package-res/resources/web/pentaho/data/_filter/_Element.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 define([
-  "pentaho/lang/Base"
+  "../../lang/Base"
```

Comment:
I don't think that the absolute ID is wrong.
My reasoning:
1. It becomes consistent with the rest of the code base (as you pointed out)
2. My IDE resolves the paths with minimal fussing of the configuration (yes, it's selfish)